examples/multiplatform-chat/src/serverJvmMain/kotlin/App.kt

```diff
@@ -43,6 +45,8 @@ fun main() {
 
     val rSocketServer = RSocketServer()
 
+    fun Payload.route(): String = metadata?.read(RoutingMetadata)?.tags?.first() ?: error("No route provided")
```

Comment:
We should put these as part of the public API. Seems here it's a local helper

Reply:
Why?
It's really local helper to get one route, for particular example. In real world, getting only first tag will not be enough I assume

Reply:
I think you gave the best example, for each domain there will be different conventions to use these.  Rather that just providing access as a dictionary, it would be nice to have extension methods you can import to make it feel natural to each domain.